package.json

```diff
@@ -261,13 +267,15 @@
     "test-integration-mssql": "cross-env DIALECT=mssql npm run test-integration",
     "test-integration-db2": "cross-env DIALECT=db2 npm run test-integration",
     "test-integration-snowflake": "cross-env DIALECT=snowflake npm run test-integration",
+    "test-integration-oracle": "cross-env LD_LIBRARY_PATH=$HOME/oracle/instantclient/ DIALECT=oracle UV_THREADPOOL_SIZE=128 npm run test-integration",
```

Comment:
I'm using GitHub Codespaces, running Node 16 and starting Oracle seems to work fine but I can't run integration tests.
```
 SequelizeConnectionError: DPI-1047: Cannot locate a 64-bit Oracle Client library: "libaio.so.1: cannot open shared object file: No such file or directory". See https://oracle.github.io/node-oracledb/INSTALL.html for help
Node-oracledb installation instructions: https://oracle.github.io/node-oracledb/INSTALL.html
You must have 64-bit Oracle Client libraries configured with ldconfig, or in LD_LIBRARY_PATH.
If you do not have Oracle Database on this computer, then install the Instant Client Basic or Basic Light package from 
https://www.oracle.com/database/technologies/instant-client/linux-x86-64-downloads.html
```

Reply:
The libaio package needs to be added manually and is not part of the instant client. 
It looks like its not installed  in VM/container? 
The instructions details are here :
https://oracle.github.io/node-oracledb/INSTALL.html (grep for libaio )